### ISSUE TITLE
Forums

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#693]
Decision on what forums are needed. Some permanent forums (a ‘queries on transcriptions’ group, for example), some temporary (e.g. to discuss Transcriber Agreement).

‘Opt in’ participation – people can select to get no email alerts, alert for each forum post, daily or weekly summary, daily or weekly compilation. Forum posts only visible once logged in – forum may be locked to specific roles or individuals (e.g. County Coordinators only).
Some forums might link well to social media – e.g. posting a question might tweet the subject line, or post to Facebook. This might be particularly good for engaging new volunteers, and promoting the resources we have available, but some monitoring would be needed, and possibly even feeding back from social media to the forums.
‘How to’ for best practice on forums, and inclusion in Codes of Conduct.
Posts listed on website in order of most recent contribution.
Number of responses to posts visible.
Responses ‘threaded’ to original post.
Response via the website or via email routed through the website.
Original poster, respondent and moderators can ‘close’ threads.

**Comment by Sherlock21** [comment]
Where will you locate the table of all these "opt in and Opt out"  features please? 
All in one please, on the User's profile page hopefully?

**Comment by PatReynolds** [comment]
Identified as urgent (needed before inviting people to sign Transcription Agreement

**Comment by richpomfret** [comment]
Currently testing/evaluating the use of Slack, Google forums and Reddit for this.

**Comment by richpomfret** [comment]
Decided to go with Google groups and have set-up a Q&A forum for existing transcribers.

**Comment by edickens** [comment]
Now assigned to me.  What do you want me to do?
My complaint about the present system of using Transcriber and Coordinator Google Groups is that they have to be managed and are not integrated with the server's UserID list.  Each time someone registers I have to go into the transcribers group and invite them to join because it is a closed group.  This is deliberate so that we do not get spam and only those people we know can post messages.
If we need to send out management messages we can do this through the server's message system, which works but has not reply or forum option.
So with this new Q&A Forum, how do people get involved with it?  Is it Closed?  Do we have to publicise the URL?  Is (will there be) reference to it on the server in the Members area?

**Comment by SteveBiggs** [comment]
Rich would like us to review the new google group "Transcription Agreement & Open Data" and add Q&A's as you feel appropriate.

**Comment by PatReynolds** [comment]
Could you particularly look at ease of using the forum _without_ logging in, and the ease of creating a log in and using the forum.  Could you try in an anonymous browser.

**Comment by SteveBiggs** [comment]
This is not a Forum, but is a Google group.
It's easy enough to view existing Q&A's anonymously, but I think you will get pushback from some who are inherently anti-Google and will not accept that they have to create a Google account to be able to post.
A true Forum is likely to get much more universal acceptance.

**Comment by PatReynolds** [comment]
I agree - something non-google would be better recieved. How about https://github.com/discourse/discourse ? We could make it the homepage of https://freeukhistory.org.uk
